packages/mongo/mongo.d.ts

```diff
@@ -107,6 +107,8 @@ export namespace Mongo {
       byteSize?: number,
       maxDocuments?: number
     ): Promise<void>;
+
+    /** @deprecated since 2.8 */
```

Comment:
After talking with @Grubba27 we would like you to add more instructions on what to use, maybe `@see createIndexAsync` or something like that. Let's use JSDocs to its full potential! 👍

Reply:
I'm fine with using `@see`, I think that is cleaner.